packages/legend-graph/src/models/GraphModifierHelper.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { GenericType } from './metamodels/pure/packageableElements/domain/GenericType';
+import type { GenericTypeReference } from './metamodels/pure/packageableElements/domain/GenericTypeReference';
+import type { Package } from './metamodels/pure/packageableElements/domain/Package';
+import type { PackageableElement } from './metamodels/pure/packageableElements/PackageableElement';
+
+// // GenericTypeReference
+export const _genericTypeReference_setValue = (
```

Comment:
I think if we already made sure we don't use these in the metamodels, it's totally harmless to move them into `DomainHelper`